src/components/Page/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { GlobalWrapper } from './styled'
+
+export const Page: React.FC = ({ children }) => {
```

Comment:
Don't need `export` here, when you export it once again below. :grin:
```suggestion
const Page: React.FC = ({ children }) => {
```